index.html

```diff
@@ -45,4 +45,91 @@ <h2 class="describe">I am a Software Developer</h2>
   
   </body>
 
+  <section class="works">
+    <h1 class="recent-works"> My Recent Works</h1>
+    <hr />
+
+    <img src="images/works.png" alt="#">
+
+    <h2 class="Stories">Multi-Post Stories</h2>
+    <p class="work-words">
+       A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+    </p>
+
+    <ul class="languages">
+       <li class="language-icons">
+           <a href="#"><img src="images/css.png" alt=""/></a>
+       </li>
+       <li class="language-icons">
+           <a href="#"><img src="images/html.png" alt=""/></a>
+       </li>
+       <li class="language-icons">
+           <a href="#"><img src="images/bootstrap.png" alt=""/></a>
+       </li>
+       <li class="language-icons">
+           <a href="#"><img src="images/ruby.png" alt=""/></a>
+       </li>
+       <br>
+    </ul>
+
+   <li class="button">
+       <a href="#"><img src="images/Enable.png" alt=""/></a>
+   </li>
+    
+ </section>
+
+ <section class="About-me">
+
+   <h1 class="about">About me</h1>
+   <p class="about-words">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+   <li class="resume-button">
+       <a href="#"><img src="images/resume.png" alt=""/></a>
+   </li>
+
+   <hr class="about-break"/>
+
+ </section>
+
+ <section class="myskills">
+
+   <ul class="language">
+       <a class="icon" href="#"><img src="images/language.png" alt=""/></a>
+       <p> Languages </p>
+   </ul>
+   <ul class="list">   
+       <a class="skillset" href="#">Javascript</a>
+       <a class="skillset" href="#">Ruby</a>
+       <a class="skillset" href="#">Html</a>
+       <a class="skillset" href="#">Css</a>
+   </ul>
+
+   <ul class="language">
+       <a class="icon" href="#"><img src="images/frameworks.png" alt=""/></a>
+       <p> Frameworks </p>
+   </ul>
+   <ul class="list">   
+       <a class="skillset" href="#">Bootstrap</a>
+       <a class="skillset" href="#">Ruby on Rails</a>
+       <a class="skillset" href="#">RSPec</a>
+       <a class="skillset" href="#">CapyBara</a>
+       <a class="skillset" href="#">Selenium</a>
+   </ul>
+
+   <ul class="language">
+       <a class="icon" href="#"><img src="images/skills.png" alt=""/></a>
+       <p> Skills </p>
+   </ul>
+   <ul class="list">   
+       <a class="skillset" href="#">Codekit</a>
+       <a class="skillset" href="#">Github</a>
+       <a class="skillset" href="#">Codepen</a>
+       <a class="skillset" href="#">Gitlab</a>
```

Comment:
- [ ]  In this section, The technology categories ("Languages", "Frameworks" and "Skills") should be in a ul, each of them in a li tag and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list ul and each of them a li. This is to make the listing orderly and semantically correct.

- [ ] Kindly remeber to align the content properly

### Your design
![image](https://user-images.githubusercontent.com/86200356/205256460-d6fa26cc-0b52-4069-8d8c-ec6632bb52e7.png)

### Figma design
![image](https://user-images.githubusercontent.com/86200356/205256593-cfbb4386-5999-424c-95a3-2871a247a599.png)